### ISSUE TITLE
Error when I try to include FRT in my project

### ISSUE THREAD

**Issue by davidc86** [sillsdev/ptx2pdf#414]
When I click PRINT, I get this: 

![image](https://user-images.githubusercontent.com/65363206/109252450-91954280-77bb-11eb-95c9-678b1f7663fe.png)

I suspect this is due to me adding/testing the FRT book in my configuration.
It has lots of unique text (much like the colophon) but includes logos and lots of stuff like tables etc. (for the full introduction, with grammar notes, explanations, etc.)

Unfortunately the archive is rather large (24MB) so it won't go here. Try: https://1drv.ms/u/s!AsIa18BQPThpr8AiBb5fpxz65GCSrw?e=8bxetk

**Comment by davidc86** [comment]
Okay, I tried the FRT book with the new version v1.5.15 and I got this error message: 

![image](https://user-images.githubusercontent.com/65363206/109823545-088c6a00-7c06-11eb-9b39-facf50ca7d6d.png)

But I'm still not able to read that message and figure out what might be wrong, except maybe I have a field marker in the FRT that is custom. (Which I do.) I have these all defined in the "custom.sty" file for Paratext, but in there I have all kinds of other specifications too, which I don't necessarily want coming into PTXprint! I was hoping to just use the Style sheet in PTXprint, so I'm not having to chase my tail all around to figure out where a layout or style problem lies. 

So, let's assume a normal MTT user will NOT use the custom.sty option (since they won't know how to edit it anyway). So if there are any custom markers in the project (made by a consultant, for example), these would likely be in the custom.sty file. In this case, the project would PASS the Basic Checks in Paratext. So the message about running the Basic Checks won't help the MTT.

Therefore, it seems reasonable to have PTXprint do a run through the entire project and compile a list of field markers and make note of any which it doesn't know about. These could be reported to the user for them to fix the problem (say there actually is bad data, and the Basic Checks would help with those), or offer to add the valid custom fields to the PTXprint Style Sheet. Then at least PTXprint would know about all the fields in the project.

**Currently, I have no real way (that I know of) of seeing all the markers in the project to know which are unspecified in the stylesheets available to PTXprint at the time I want to print something.**  Is their such a feature that I'm not seeing? Maybe at the bottom of the Style sheet?? I do recall there is some residue at the bottom of the list.

Okay, I found the problem: I had a custom **\zhd** field in the FRT book. This was apparently a field to mark "pagebreak". But I found in the USFM PDF file the preferred field marker **\pd** for this. So I replaced them all. And reran the PRINT. Now I get this:

![image](https://user-images.githubusercontent.com/65363206/109827277-9453c580-7c09-11eb-80db-c1dad1b3eb0f.png)

Which I think is saying I haven't defined the EN and IN versions of the colophon copyright text. So, I think I need to figure that out...
